inc/extras.php

```diff
@@ -89,3 +89,13 @@ function twentyseventeen_is_frontpage() {
 		return false;
 	endif;
 }
+
+/**
+ * Add a pingback url auto-discovery header for singularly identifiable articles.
+ */
+function twentyseventeen_pingback_header() {
+	if ( is_singular() && pings_open() ) {
+		echo '<link rel="pingback" href="', bloginfo( 'pingback_url' ), '">' . "\n";
```

Comment:
Well, with the `\n` it properly adds a newline so the HTML output looks cleaner.

I rather wonder about the commas there.